content/guides/guides/install.md

```diff
@@ -93,10 +93,16 @@ Congratulations! You now have a working IPFS installation on your computer.
 `ipfs-update` can be downloaded for your platform at: https://dist.ipfs.io/#ipfs-update
 
 If you have a working Go environment (>=1.12), you can also install it with:
+
 ```
-$ go get -u github.com/ipfs/ipfs-update
+# You need to CD to a directory _outside_ of your GOPATH.
+$ cd /
+# Install with go modules enabled
+$ GO111MODULE=on go get -u github.com/ipfs/ipfs-update
```

Comment:
Given that we're using go modules, we shouldn't use the `-u` flag.

Reply:
Updated. LGTY now?